app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Person.cs

```diff
@@ -17,13 +17,22 @@ public class Person
         public string Gender { get; set; }
         public bool? DateDeathConfirmed { get; set; }
         public string Incacerated { get; set; }
+        //the unit is centimeter
+        public string Height { get; set; }
+        //the unit is pounds
```

Comment:
can we add this using the description tag so it can be included in documentation For eg  [Description("Height is in centimers")]